### ISSUE TITLE
Timelines ignore tags

### ISSUE THREAD

**Issue by mellypop** [April-Gras/obsidian-auto-timelines#3]
I am trying to render two distinct timelines, but the events I create are showing up on both timelines, regardless of which timelines are tagged, and I'm not sure what I'm doing wrong. For now, I have three events:

```yaml
aat-event-start-date: 1
aat-event-end-date: 316
aat-render-enabled: true
timelines: ['complete']
```

```yaml
aat-event-start-date: 317
aat-event-end-date: true
aat-render-enabled: true
timelines: ['complete']
```

```yaml
aat-event-start-date: 317
aat-event-end-date: 317
aat-render-enabled: true
timelines: ['brightwind']
```

And I have two timelines that look like these (without the leading backslashes):

```
\```aat-vertical
complete
\```
```
```
\```aat-vertical
brightwind
\```
```

**Comment by ObsidianTTRPGProject** [comment]
Can confirm I am experiencing this issue also. 
![image](https://user-images.githubusercontent.com/96321415/236591515-19f99a97-c95e-43b8-bef9-49ed01ba0084.png)
![image](https://user-images.githubusercontent.com/96321415/236591532-2eae4e11-b483-4a52-9709-e0ffb8629f72.png)
![image](https://user-images.githubusercontent.com/96321415/236591539-e2983155-98ee-40d3-8660-109ca861a982.png)
![image](https://user-images.githubusercontent.com/96321415/236591541-7d1a9fdc-0d6e-4531-a4ec-ac17d1baae8a.png)
![image](https://user-images.githubusercontent.com/96321415/236591553-3eb83346-3e2c-4d59-a9cf-4b4808b38bbe.png)

**Comment by ObsidianTTRPGProject** [comment]
@April-Gras my timelines are not rendering at all now.

**Comment by April-Gras** [comment]
> @April-Gras my timelines are not rendering at all now.

Indeed this has been addressed in #6 and should probably be fixed if you update to [0.0.6](https://github.com/April-Gras/obsidian-auto-timelines/releases/tag/0.0.6)
